cli/cmd/generate_execute.go

```diff
@@ -0,0 +1,392 @@
+			resourceDelete += 1
+		case c.Change.Actions.Update():
+			resourceUpdate += 1
+		case c.Change.Actions.Replace():
+			resourceReplace += 1
+		}
+	}
+
+	return &tfPlanChangesSummary{
+		plan:    plan,
+		create:  resourceCreate,
+		deleted: resourceDelete,
+		update:  resourceDelete,
+		replace: resourceReplace,
+	}, nil
+}
+
+func TerraformInit(tf *tfexec.Terraform) error {
+	cli.StartProgress("Running terraform init")
+	if err := tf.Init(context.Background(), tfexec.Upgrade(true)); err != nil {
+		return errors.Wrap(err, "failed to execute terraform init")
+	}
+	cli.StopProgress()
+
+	return nil
+}
+
+// Run terraform plan using the workingDir from *tfexec.Terraform
+//
+// - Run plan
+// - Get plan file details (returned)
+func TerraformExecPlan(tf *tfexec.Terraform) (*tfPlanChangesSummary, error) {
+	// Plan
+	cli.StartProgress("Running terraform plan")
+	if _, err := tf.Plan(context.Background(), tfexec.Out("tfplan.json")); err != nil {
+		return nil, err
+	}
+	cli.StopProgress()
+
+	// Gather changes from plan
+	return processTfPlanChangesSummary(tf)
+}
+
+// Run terraform apply using the workingDir from *tfexec.Terraform
+//
+// - Run plan
+// - Get plan file details (returned)
+func TerraformExecApply(tf *tfexec.Terraform) error {
+	// Plan
+	cli.StartProgress("Running terraform apply")
+	if err := tf.Apply(context.Background()); err != nil {
+		return err
+	}
+	cli.StopProgress()
+
+	return nil
+}
+
+// Simple helper to prompt for next steps after TF plan
+func promptForTerraformNextSteps(previewShown *bool, data tfPlanChangesSummary) (int, error) {
+	options := []string{
+		"Continue with Terraform Apply",
+	}
+
+	// Omit option to show details if we already have
+	if !*previewShown {
+		options = append(options, "Show details")
+	}
+	options = append(options, "Quit")
+
+	var answer int
+	err := SurveyQuestionInteractiveOnly(SurveyQuestionWithValidationArgs{
+		Prompt: &survey.Select{
+			Message: fmt.Sprintf(
+				"Terraform will create %d resources, delete %d resources, update %d resources, and replace %d resources.",
+				data.create,
+				data.deleted,
+				data.update,
+				data.replace),
+			Options: options,
+		},
+		Response: &answer,
+	})
+
+	return answer, err
+}
+
+// this helper function is called when terraform flow has been completely executed through apply
+func provideGuidanceAfterSuccess(workingDir string, laceworkProfile string) {
+	out := new(strings.Builder)
+	fmt.Fprintf(out, "Lacework integration was successful! Terraform code saved in %s.\n\n", workingDir)
+	fmt.Fprintln(out, "Use the Lacework CLI to view integration status:")
+
+	laceworkCmd := "  lacework integration list\n\n"
+	if laceworkProfile != "" {
+		laceworkCmd = fmt.Sprintf("  lacework -p %s integration list\n\n", laceworkProfile)
+	}
+	fmt.Fprint(out, laceworkCmd)
+
+	cli.OutputHuman(out.String())
+}
+
+// this helper function is called when existing the Terraform flow without applying to provide guidance on next steps
```

Comment:
Is this comment missing some words, I don't think it quite makes sense

Reply:
lol; will fix